dv/uvm/core_ibex/scripts/compile-tb.py

```diff
@@ -49,7 +85,8 @@ def main() -> int:
                              'out': args.output,
                              'cmp_opts': get_compile_opts(args.ibex_config,
                                                           args.simulator)
-                         })
+                         } | cosim_subst_vars
+                         )
```

Comment:
Ha! As Canberk and I discovered a couple of days ago, this operator was only added in Python 3.9. Since we only require Python 3.6, you'll have to do the ugly stateful update. (But you can, at least, hoist it out of the loop)